tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py27-cl
+envlist = py27, py27-cl, py35, py35-cl
```

Comment:
Let's not add py35-cl for now. Once compare-locales is up to snuff re 3.5, we'll likely find issues in the migration code that are py2-specific.